tests/foreman/cli/test_satellitesync.py

```diff
@@ -230,20 +400,22 @@ def class_export_entities():
 
 
 @pytest.fixture(scope='function')
-def function_export_cv_directory():
-    """Create directory for CV export and at the end remove it"""
-    export_dir = f'{get_export_base()}/{gen_string("alpha")}'
-    ssh.command(f'mkdir {export_dir}')
+def export_cv_dir(default_sat, module_org):
+    """Get directory for CV export and at the end remove it"""
+    export_dir = '/var/lib/pulp/exports/'
     yield export_dir
     # Deletes directory created for CV export Test
-    ssh.command(f'rm -rf {export_dir}')
+    default_sat.execute(f'rm -rf {export_dir}/{module_org.name}')
```

Comment:
there's an extra `/` in the path here. It shouldn't negatively effect the behavior, but there are more consistent ways to construct these paths using `pathlib.PurePath` instances.